nearlib/test/integration.test.js

```diff
@@ -43,6 +49,37 @@ test('create account and then view account returns the created account', async (
     fail('exceeded number of retries for viewing account');
 });
 
+test('deploy contract and make function calls', async () => {
+    var data = [...fs.readFileSync('../tests/hello.wasm')];  
```

Comment:
nit: that's work for linter (which I plan to setup), but generally use `let` in place of `var` for more sane scoping (`var` has function scope instead of block scope).

Reply:
in any case this probably should be const